content/200-orm/100-prisma-schema/20-data-model/40-views.mdx

```diff
@@ -366,6 +366,6 @@ Some databases support materialized views, e.g. [PostgreSQL](https://www.postgre
 
 Materialized views persist the result of the view query for faster access and only update it on demand.
 
-Currently Prisma ORM has no understanding of materialized views, but when you [manually create a view](#create-a-view-in-the-underlying-database) you can also create a materialized view by using the corresponding command in the underlying database. You can then use Prisma Client's [raw query functionality](/orm/prisma-client/queries/raw-database-access) to execute the command to refresh the view manually.
+Currently Prisma ORM has no understanding of materialized views, but when you [manually create a view](#create-a-view-in-the-underlying-database) you can also create a materialized view by using the corresponding command in the underlying database. You can then use Prisma Client's [TypedSQL functionality](/orm/prisma-client/using-raw-sql) to execute the command to refresh the view manually.
```

Comment:
```suggestion
Currently, Prisma ORM does not support materialized views. However, when you [manually create a view](#create-a-view-in-the-underlying-database), you can also create a materialized view with the corresponding command in the underlying database. You can then use Prisma Client's [TypedSQL functionality](/orm/prisma-client/using-raw-sql) to execute the command and refresh the view manually.
```

Also can we add an example here? This can be done post-GA too. @mhessdev WDYT?

Reply:
Yup i like that language better. @jharrell